webserver/cWebem.cpp

```diff
@@ -199,70 +180,6 @@ namespace http {
 				(int)myPages.size(), (int)myActions.size(), (int)myWhitelistURLs.size(), (int)myWhitelistCommands.size());
```

Comment:
Perhaps we should use %zu so we do not have to cast to an integer?

Reply:
done